doc/source/info_file.rst

```diff
@@ -3,19 +3,17 @@
 The ``info.yaml`` file
 ======================
 
-At the heart of Papis there is the information file. The info file contains
-all information about the documents.
+At the heart of Papis is the information file. The info file contains all
+information about the documents.
 
-It uses the `YAML <https://yaml.org>`__ syntax to store
-information, which is a very human-readable language.
-It is quite format-free:
-`papis` does not assume that any special information should be there.
-However it will interpret the field ``files`` as the files linked to the
-document for the ``papis open`` command. The ``files`` field
-should be formatted as a YAML list.
+It uses the `YAML <https://yaml.org>`__ syntax to store information, which is a
+very human-readable language. It is quite format-free: Papis does not assume
+that any special information should be there. However it will interpret the
+field ``files`` as the files linked to the document for the ``papis open``
+command. The ``files`` field should be formatted as a YAML list.
```

Comment:
```
It uses the `YAML <https://yaml.org>`__ syntax, which is very human-readable.
It's quite format-free: Papis does not assume that any specific information is
contained in it. However, Papis will interpret the field ``files`` as specifying the
files linked to the document and opens them with ``papis open``. The ``files``
field should be formatted as a YAML list.
```
Should we mention `papis_id`, which is required (though autocreated)? We could also mention the fields required by `papis doctor` for a given bibtex type.

Reply:
Yeah, mentioning `papis_id` is probably a good idea because that's required by the caching / database parts which are always on. For the rest of the keys, we could say something like "various commands may require additional keys (e.g. the `files` key is required by `papis open`)"?